opensuse/cpe/opensuse-cpe-dictionary.xml

```diff
@@ -2,19 +2,24 @@
 <cpe-list xmlns="http://cpe.mitre.org/dictionary/2.0"
           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
           xsi:schemaLocation="http://cpe.mitre.org/dictionary/2.0 http://cpe.mitre.org/files/cpe-dictionary_2.1.xsd">
+      <cpe-item name="cpe:/o:opensuse">
+            <title xml:lang="en-us">openSUSE</title>
+            <!-- the check references an OVAL file that contains an inventory definition -->
+            <check system="http://oval.mitre.org/XMLSchema/oval-definitions-5" href="filename">installed_OS_is_opensuse</check>
+      </cpe-item>
       <cpe-item name="cpe:/o:opensuse:leap:42.1">
             <title xml:lang="en-us">openSUSE Leap 42.1</title>
             <!-- the check references an OVAL file that contains an inventory definition -->
-            <check system="http://oval.mitre.org/XMLSchema/oval-definitions-5" href="filename">installed_OS_is_opensuse</check>
+            <check system="http://oval.mitre.org/XMLSchema/oval-definitions-5" href="filename">installed_OS_is_opensuse421</check>
```

Comment:
@abergmann why this change? This is adds a lot of maintenance for upstream activities. In addition, the change doesn't really need to happen if you are checking for any version of openSUSE 42